components/input/Input.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react"
+import { cn } from "../../lib/utils"
+
+export interface InputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {}
+
+const Input = React.forwardRef<HTMLInputElement, InputProps>(
+  ({ placeholder, className, type, ...props }, ref) => {
+    return (
+      <input
+        type={type}
+        placeholder={placeholder}
```

Comment:
Don't need to destructure these or do this, since we're already spreading `{...props}` below

Reply:
Thanks Mike! `type` and `placeholder` removed